glide.yaml

```diff
@@ -6,181 +6,133 @@ excludeDirs:
 - openshift.local.etcd
 - openshift.local.volumes
 import:
-# kube first
 - package: k8s.io/kube-openapi
   version: 39a7bf85c140f972372c2a0d1ee40adbf0c8bfe1
 - package: k8s.io/code-generator
-  repo:    git@github.com:openshift/kubernetes-code-generator
   version: release-1.9.1
+  repo: git@github.com:openshift/kubernetes-code-generator
 - package: k8s.io/apimachinery
-  repo:    git@github.com:openshift/kubernetes-apimachinery
   version: release-1.9.1
+  repo: git@github.com:openshift/kubernetes-apimachinery
 - package: k8s.io/api
-  repo:    git@github.com:openshift/kubernetes-api
   version: release-1.9.1
+  repo: git@github.com:openshift/kubernetes-api
 - package: k8s.io/client-go
-  repo:    git@github.com:openshift/kubernetes-client-go
   version: release-1.9.1
+  repo: git@github.com:openshift/kubernetes-client-go
 - package: k8s.io/metrics
-  repo:    git@github.com:openshift/kubernetes-metrics
   version: release-1.9.1
+  repo: git@github.com:openshift/kubernetes-metrics
 - package: k8s.io/apiserver
-  repo:    git@github.com:openshift/kubernetes-apiserver
   version: release-1.9.1
+  repo: git@github.com:openshift/kubernetes-apiserver
```

Comment:
Please undo this reformatting.

Reply:
That's all done by glide. I couldn't run glide on my local repo - it doesn't run for me, times out all the time, so @rajatchopra  ran it for me  and he didn't do anything other than `glide get` && `./hack/update-deps.sh`.  Is something that's glide version specific causing this?

Reply:
I’ve never had glide time out - are you on an old version?  Bad internet?

Reply:
glide version is v0.13.1  and its a fast[ish] pipe - 200mb+ consistently (and also tried it on a T3 connection) - so i suspect its not the speed. Plus did try it on 3 different vms as well (all fedora 27 though), so something else. Anyway, will look at it at some other time.